src/config/botConfig.js

```diff
@@ -1,15 +1,8 @@
 /* External modules */
 const TelegramBot = require('node-telegram-bot-api')
+const { token:potatoe } = require('./botTokens.js')
```

Comment:
This could be renamed to `token` and be separated as a single string, like:

```
const token = require('./botToken')
```

If any new property appears, we can change it to an object. What do you think?